launcher/sdw_updater_gui/strings.py

```diff
@@ -1,56 +1,41 @@
-# checking for updates
-description_status_checking_updates = (
-    "<h3> Checking for updates... </h3> <p>This should take around 3 to 5 minutes.</p>"
+headline_introduction = "Preflight security updates"
+description_introduction = (
```

Comment:
FYI the wording on that screen is based on the designs in https://github.com/freedomofpress/securedrop-workstation/issues/478#issuecomment-612518442 and some further discussion Nina and I had yesterday. For the heading I went with "Preflight security updates" because I felt it's important to include the word "security" there, but happy to continue to discuss.